presto-docs/src/main/sphinx/security/salesforce.rst

```diff
@@ -0,0 +1,80 @@
+============================
```

Comment:
Make these bars the same length as the text

Reply:
Gotcha.